### DISCUSSION TITLE
No apps are updating in AppCentre

### DISCUSSION THREAD

**Discussion by Oberynnavalny** [elementary/appcenter#2222] [Q&A]
All the apps install quietly, however when I click to update them the same error comes out, which is “Aborted due to an error (While applying additional data: the apply_extra script terminated with error 256)””. In principle, the problem is not terrible with applications, but this error occurs even with runtime updates. I have torn down Elementary Os several times and the problem still persists

**Comment by GianniJT** [comment]
I'm having the same issue.  I just installed v.8 and the same error shows up when trying to install runtime updates.

**Comment by teamcons** [comment]
In a terminal
flatpak repair
Then
flatpak update

The issue is caused by some canonical hijinx im Ubuntu iirc
